### PR TITLE
Remove pry requirement, improve readme, add ENV variable support

### PR DESCRIPTION
**Pull request by DerekMaffett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,26 +9,65 @@ cannot find it out in the environment.
...
+          <param name="teamcity.step.mode" value="default" />
+        </parameters>
+      </runner>
+    </build-runners>
+    <requirements />
+  </settings>
+</meta-runner>
+```
+
+The previous steps all rely on bot accounts - set up a TeamCity account with
```
Comment (aaronjensen) [inline README.md]: may be nice to have this step before the previous steps... set up a bot account/identify the account you want to send messages, etc.

File: README.md [hunk 2]
```diff
@@ -9,26 +9,65 @@ cannot find it out in the environment.
...
+3. On TeamCity, set up your build configuration steps to run `bundle install`
+   initially. Make a final step to trigger regardless of previous step outcomes
+   and use the following as a meta-runner: 
 
-1. Fork it ( http://github.com/<my-github-username>/flowdock_notify/fork )
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+```xml
```
Comment (aaronjensen) [inline README.md]: indent this 4 spaces so it is part of step 3

File: lib/flowdock_build_notifier/configuration.rb [hunk 3]
```diff
@@ -14,13 +14,14 @@ def self.load
...
+      @flowdock_user_token = ENV['FLOWDOCK_USER_TOKEN'] || config.fetch('flowdock_user_token')
       @flow_name= config.fetch('flow_name')
+
       @email_map = config.fetch('email_map') { {} }
 
       @teamcity_url = config.fetch('teamcity_url')
-      @teamcity_user = config.fetch('teamcity_user')
-      @teamcity_password = config.fetch('teamcity_password')
+      @teamcity_user = ENV['FLOWDOCK_NOTIFIER_TEAMCITY_USER'] || config.fetch('teamcity_user')
+      @teamcity_password = ENV['FLOWDOCK_NOTIFIER_TEAMCITY_PASSWORD'] || config.fetch('teamcity_password')
```
Comment (aaronjensen) [inline lib/flowdock_build_notifier/configuration.rb:24]: :+1:

File: README.md [hunk 4]
```diff
@@ -9,26 +9,65 @@ cannot find it out in the environment.
...
+```
 
-TODO: Write usage instructions here
+The email map is used to connect the author of the last commit to the
+corresponding Flowdock account.
 
-## Contributing
+3. On TeamCity, set up your build configuration steps to run `bundle install`
+   initially. Make a final step to trigger regardless of previous step outcomes
+   and use the following as a meta-runner:
```
Comment (aaronjensen) [inline README.md]: this is kinda "huh?" so it may be good to add screenshots or a link to what a meta-runner is or how to set it up

Comment (DerekMaffett) [inline README.md]: :green_heart: a7ac0c9

**Comment by aaronjensen** [comment]
Nice, thanks!

**Pull request merged**